lib/chat_api_web/channels/conversation_channel.ex

```diff
@@ -80,20 +81,33 @@ defmodule ChatApiWeb.ConversationChannel do
     {:reply, {:ok, payload}, socket}
   end
 
+  @impl true
+  def handle_in("shout", payload, socket) do
+    Logger.warn(
+      "'shout' is deprecated as event name on a new message and will be removed in a future version. Please migrate to a newer version of a client."
+    )
+
+    handle_in_msg("shout", payload, socket)
+  end
+
+  def handle_in("message:created", payload, socket) do
+    handle_in_msg("message:created", payload, socket)
+  end
+
   # It is also common to receive messages from the client and
   # broadcast to everyone in the current topic (conversation:lobby).
-  def handle_in("shout", payload, socket) do
+  defp handle_in_msg(event_name, payload, socket) do
```

Comment:
nitpick -- can we name this something like `handle_incoming_message` instead? i think it's a bit more explicit :) 

also let's move this to the bottom with the other private methods so we can fix the warning below 👍

Reply:
I renamed the method and moved it to the bottom - naming is hard ;-)

I also made similar changes in `notification_channel`.